apps/aeutil/lib/serialization.ex

```diff
@@ -721,7 +721,7 @@ defmodule Aeutil.Serialization do
   def tag_to_type(24), do: OracleResponseTx
   def tag_to_type(25), do: OracleExtendTx
   def tag_to_type(30), do: Name
-  def tag_to_type(31), do: NameCommitmentTx
+  def tag_to_type(31), do: NameCommitment
```

Comment:
is this changed on purpose?

Reply:
Yes, because was not working. Have to be like now